homeassistant/components/songpal/config_flow.py

```diff
@@ -116,7 +116,7 @@ async def async_step_ssdp(
         ]
 
         # Ignore Bravia TVs
-        if "videoScreen" in service_types:
+        if "videoScreen" in service_types or "video" in service_types:
```

Comment:
is service_types a string? In that case you can remove the first option because `video` is also contained in `videoScreen`

Reply:
Looking at tests and [this comment](https://github.com/home-assistant/core/issues/133689#issuecomment-2557921344), it seems to be a list: https://github.com/home-assistant/core/blob/67f0de441b489890efa802a325f187b761098ad6/tests/components/braviatv/test_config_flow.py#L58-L65

Reply:
Yeah, it's a list indeed. Some newer bravias (https://github.com/home-assistant/core/issues/133689#issuecomment-2558122304) apparently do not have `videoScreen` anymore, hence the adjustment.

In the worst case, some supported devices won't be discovered automatically, but the known supported soundbars (based on grep 'video' on https://github.com/rytilahti/python-songpal/tree/master/devinfos) do not seem to expose neither, so I think we can risk false negatives here for better UX for TV users.

Reply:
Would be nicer if it was a `set` so avoid the linear search but its so short it won't matter so much in practice.